src/NHibernate/Logging.cs

```diff
@@ -250,12 +261,12 @@ public LegacyLoggerFactoryAdaptor(ILoggerFactory factory)
 				_factory = factory;
 			}
 
-			public INHibernateLogger LoggerFor(string keyName)
+			INHibernateLogger INHibernateLoggerFactory.LoggerFor(string keyName)
```

Comment:
Just avoids a R# warning about hiding the `LoggerFor` of the encapsulating class.